x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -286,63 +279,26 @@ const TimelineDescriptionComponent: React.FC<FlyoutHeaderProps> = ({ timelineId
   );
   const dispatch = useDispatch();
 
-  const onReadMore: MouseEventHandler<HTMLButtonElement> = useCallback(
-    (event: MouseEvent<HTMLButtonElement>) => {
-      dispatch(
-        setActiveTabTimeline({
-          id: timelineId,
-          activeTab: TimelineTabs.notes,
-          scrollToTop: true,
-        })
-      );
-    },
-    [dispatch, timelineId]
-  );
+  const onReadMore = useCallback(() => {
+    dispatch(
+      setActiveTabTimeline({
+        id: timelineId,
+        activeTab: TimelineTabs.notes,
+        scrollToTop: true,
+      })
+    );
+  }, [dispatch, timelineId]);
 
-  return (
+  const hasDescription = !!description;
+  return hasDescription ? (
     <EuiText size="s" data-test-subj="timeline-description">
-      <ReadMoreButton description={description || commonI18n.DESCRIPTION} onclick={onReadMore} />
+      <ReadMoreButton description={description} onclick={onReadMore} />
     </EuiText>
-  );
+  ) : null;
 };
 
 const TimelineDescription = React.memo(TimelineDescriptionComponent);
 
-const TimelineStatusInfoComponent: React.FC<FlyoutHeaderProps> = ({ timelineId }) => {
```

Comment:
Yea, can we keep this component and just use the text `saved` and if there are `changes` have the text 'Unsaved Changes' or something similar?

Reply:
That's a great idea. Right now it's shown in the button tooltip which isn't the best place imo. Plus, having it only visible on hover, causes all sorts of issues in unit testing and tends to be flaky in integration testing.